mesheryctl/internal/cli/root/app/view_test.go

```diff
@@ -92,7 +93,7 @@ func TestAppView(t *testing.T) {
 			apiResponse := utils.NewGoldenFile(t, tt.Fixture, fixturesDir).Load()
 
 			// set token
-			tokenPath = tt.Token
+			constants.TokenFlag = tt.Token
```

Comment:
Does it make sense to add this flag under a different package or import the constant package with some other alias? This sounds like we are modifying the constants.

Reply:
So the new logic of creating a http request does two things
1. search Token passed by `--token` that has been assigned to var constants.TokenFlag
2. If found ""; i.e. no token was passed with flag, location is fetched from meshconfig
3. Checks if the file exists
4. Try adding the file to request

returns the http request.

During unit tests, we mock server response but token needs to be attached to due to the requirement of the logic, so we save a sample token but it doesn't exists at the location defined in the meshconfig, so two things could be done here; pass the token location with "--token" flag or set the value of `constants.TokenFlag` manually, and passing the flag does the same things so chose wrote it manually.

Reply:
@piyushsingariya, I understand the reasons behind, manually storing tt.Token value in constants.TokenFlag
The variables/flags defined in constants.go must be used for read only purpose. 
Ideally, any flag/field which can be modified later, should belong to a different package. 
May be the TokenFlag field should move to either internal/cli/root/config package or pkg/utils package.

Reply:
makes sense, moved to utils package